recipes-devtools/sam-ba/sam-ba_3.2.3.bb

```diff
@@ -8,13 +8,23 @@ SRC_URI = "https://github.com/atmelcorp/${BPN}/releases/download/v${PV}/${BPN}_$
 SRC_URI[md5sum] = "11746fcc1f97e054fa6a8c9c57c12a20"
 SRC_URI[sha256sum] = "3ecd53dab9a60debe5a861e272e095da008300c860ce7494102ad987eea7c8e9"
 
+inherit deploy
+
 S = "${WORKDIR}/${BPN}_${PV}"
 
 do_install () {
     install -d ${D}${bindir}/sam-ba_cdc_linux
     cp -R --no-dereference --preserve=mode,links ${S}/* ${D}${bindir}/sam-ba_cdc_linux
 }
 
+addtask deploy after do_install
+
+do_deploy () {
+        #echo "Copying ${fit_image_basename}.itb and source file to ${DEPLOYDIR}..."
```

Comment:
I don't think ${fit_image_basename} is defined here. Should this message be removed entirely?

Reply:
Yes I think we can drop it it seems unused

Reply:
Ok, thank you. Will you update this PR or do you want me to do it?

Reply:
I am traveling today so if you can do it for me that would be fine otherwise I would be able to get it done tomorrow at the latest

Reply:
I don't have a way of updating this PR without committing to yoe/master, so please be so kind and make this change.

Thank you,
Razvan